docs/README.md

```diff
@@ -295,6 +295,10 @@ import x
 
 ```
 
+## `.gitignore`
+
+To skip `.gitignore` glob patterns use `--gitignore` flag.
```

Comment:
Can you specify that this feature is optional, if they want to use this feature, let's write that they need to download the **pathspec** library? As we did in **pyproject.toml** support.

Reply:
I got your point!
I'm gonna work on that!